apps/authentication/mfa/sms.py

```diff
@@ -14,8 +15,13 @@ class MFASms(BaseMFA):
 
     def __init__(self, user):
         super().__init__(user)
-        phone = user.phone if self.is_authenticated() else ''
-        self.sms = SendAndVerifyCodeUtil(phone, backend=self.name)
+        phone, user_info = '', None
+        if self.is_authenticated():
+            phone = user.phone
+            user_info = SmsUserSerializer(user).data
+        self.sms = SendAndVerifyCodeUtil(
+            phone, backend=self.name, user_info=user_info
+        )
 
     def check_code(self, code):
         assert self.is_authenticated()
```

Comment:
There aren't any significant changes to the code compared with its previous version in 2021-09-01 that would cause it to behave in an unexpected way today in 2024-11-20.

It is recommended to ensure you have updated all required dependencies since this doesn't show up in the provided data.

However, no major problems were found during my review. If the implementation continues to work as intended without these issues arising again, we don't need any further adjustments for now. 

If there's something specific you want us to do regarding your project, then let me know! For instance:

 - We can adjust test coverage of the current function;
 - Improve logging levels (if any logging happens);
 - Refine docstrings or improve readability;

The above suggestions are based on our understanding at this time about the existing structure of code and how Python works currently under the hood.